src/templates/Layout.tsx

```diff
@@ -38,11 +38,16 @@ export const Layout = ({
   className?: string;
   children: ReactNode;
 }): JSX.Element => {
+  const { renderContext } = useServerContext();
   return (
     <>
       <HtmlHead>{head}</HtmlHead>
       <body>
-        <AbsoluteArea name="navArea" allowedTypes={["luxe:navMenu"]} numberOfItems={1} />
+        <AbsoluteArea
+          name="nav"
+          parent={renderContext.getSite().getHome()}
+          nodeType="luxe:navMenu2"
```

Comment:
minor: I'm never a big fan of those suffixes, can we use something like `luxe:navigationMenu` or `luxe:navAreaMenu`? 🙂

Reply:
There's no versioning in CND, I never know how to do it. Let's go for `navigationMenu`

Reply:
agreed, that's a recurring problem 😕 
Which reminds me that we should implement a way to force the deployment of a JS module, even if it contains breaking changes. That would be very handy during development